docs/backend_guide/vllm.rst

```diff
@@ -1,3 +1,30 @@
+..
```

Comment:
Might be good to add support for RST files in the [copyright hook](https://github.com/triton-inference-server/server/blob/main/tools/add_copyright.py#L247). Should be as simple as:
```py
@register(has_ext([".rst"]))
def rst(path):
    update_or_add_header(path, prefix_lines(LICENSE_TEXT, ".. "))
```